website/statistics.py

```diff
@@ -504,6 +507,25 @@ def _determine_bool(bool_str):
     return False
 
 
+def _args_checks():
```

Comment:
I think we should maybe make this name a bit more specific, to state which args we're checking exactly. Something like `_get_args_live_stats()` or `_get_show_collapse_args()` ?